networks/algorand/rpc.csv

```diff
@@ -0,0 +1,57 @@
+getblock-mainnet-free,GetBlock,Free,Resent-State,mainnet,$0,$0,99.90%,NULL,NULL,FALSE,"[""Algorand Native API"",""Indexer API"",""Algo API""]","[""50K CU"",""5 RPS"",""2 Access Tokens""]","[""Privacy Protected Relays"",""Transaction Sanitizers"",""Dedicated nodes with enterprise-grade security"",""Multi-region infrastructure""]","[""Real-time performance tracking"",""Algorand Metrics Dashboard""]","[""Global""]",NULL,NULL,NULL,FALSE,"[""[Website](https://getblock.io/nodes/algo/)"",""[Docs](https://docs.getblock.io/?q=algorand)""]",,
+getblock-mainnet-pay-as-you-go,GetBlock,Pay-as-you-go,Resent-State,mainnet,$49/$199/$499,$0.98/$0.9/$0.83/1M CU,99.90%,NULL,NULL,FALSE,"[""Algorand Native API"",""Indexer API"",""Algo API""]","[""50M/220M/600M CU"",""25/100/200 RPS"",""10/25/50 Access Tokens""]","[""Privacy Protected Relays"",""Transaction Sanitizers"",""Dedicated nodes with enterprise-grade security"",""Multi-region infrastructure""]","[""Real-time performance tracking"",""Algorand Metrics Dashboard""]","[""Global""]",NULL,NULL,NULL,FALSE,"[""[Buy](https://getblock.io/pricing/)"",""[Docs](https://docs.getblock.io/?q=algorand)""]",,
+getblock-mainnet-enterprise,GetBlock,Enterprise,Resent-State,mainnet,Up to $999,Custom,99.90%,NULL,NULL,FALSE,"[""Algorand Native API"",""Indexer API"",""Algo API""]","[""custom limits""]","[""Privacy Protected Relays"",""Transaction Sanitizers"",""Dedicated nodes with enterprise-grade security"",""Multi-region infrastructure""]","[""Real-time performance tracking"",""Algorand Metrics Dashboard"",""Advanced statistics""]","[""Global""]",NULL,NULL,NULL,FALSE,"[""[Contact](https://account.getblock.io/sign-up)"",""[Docs](https://docs.getblock.io/?q=algorand)""]",,
+getblock-mainnet-dedicated,GetBlock,Dedicated,Resent-State,mainnet,Up to $1K,Custom,99.90%,NULL,NULL,FALSE,"[""Algorand Native API"",""Indexer API"",""Algo API""]","[""custom limits"",""unlimited requests"",""unlimited rate limit""]","[""Privacy Protected Relays"",""Transaction Sanitizers"",""Dedicated nodes with enterprise-grade security"",""Multi-region infrastructure""]","[""Real-time performance tracking"",""Algorand Metrics Dashboard""]","[""Global""]",NULL,NULL,NULL,FALSE,"[""[Contact](https://account.getblock.io/sign-up)"",""[Docs](https://docs.getblock.io/?q=algorand)""]",,
+getblock-mainnet-pay-as-you-go-archive,GetBlock,Pay-as-you-go,Full Archive,mainnet,$49/$199/$499,$0.98/$0.9/$0.83/1M CU,99.90%,NULL,NULL,FALSE,"[""Algorand Native API"",""Indexer API"",""Algo API""]","[""50M/220M/600M CU"",""25/100/200 RPS"",""10/25/50 Access Tokens""]","[""Privacy Protected Relays"",""Transaction Sanitizers"",""Dedicated nodes with enterprise-grade security"",""Multi-region infrastructure""]","[""Real-time performance tracking"",""Algorand Metrics Dashboard""]","[""Global""]",NULL,NULL,NULL,FALSE,"[""[Buy](https://getblock.io/pricing/)"",""[Docs](https://docs.getblock.io/?q=algorand)""]",,
```

Comment:
I think it is not clear for the end consumer to see $0.98/$0.9/$0.83/1M CU as there are no delimeter difference between price options and for how much you are buying something.
I mean, maybe we use `$0.98/$0.9/$0.83 per 1M CU` or something like that?